src/schemas/xspec.rnc

```diff
@@ -56,7 +56,7 @@ scenario =
 	## If any scenario has a focus attribute, any scenarios without a focus 
 	## attribute will be classed as pending.
 	element scenario { common-scenario-attributes, shared?,
-		label,
+		label, variable*,
```

Comment:
Is this `variable*` necessary? Removing it does not seem to affect *.xspec files added or modified by this pull request.

Reply:
Not necessary. For my planned scope of work, it's redundant with the first `variable*` in `matching-scenario`, `function-scenario`, and `named-scenario`. I will remove `variable*` from this line. I wasn't intending to alter the schema or behavior for Schematron tests here.

Reply:
Covered in 988163008627a4fa970b44253ec9cb70ec1ef139